lang/lang_en_fr.txt

```diff
@@ -930,7 +922,7 @@
 "Prusa i3 MK3S OK."
 "\x00"
 
-#MSG_CALIBRATE_BED_RESET
+#MSG_CALIBRATE_BED_RESET c=18
 "Reset XYZ calibr."
 "Reinit. calib. XYZ"
 
```

Comment:
"reprise" it's more appropriate. It's like taking it back or continuing.

Reply:
In this context, I read `Reprise` as "take back", as in stop the print, more than "continue".

Reply:
sorry google translate

Reply:
Oh, I see your edit to your previous comment. As we don't have enough space to indicate that it's the noun form of `reprise` we're using, not the verb form, `reprise` probably isn't the best choice. `Reprend.` still makes sense, and I think is clearer.

Reply:
just for example (Tomorrow I have to go back to work.):
"demain je dois reprendre le travail" or "demain je reprends le travail" = "demain reprise du travail" or "demain c'est la reprise du travail"

Reply:
Which one should I use?

Reply:
```
Reprise impression (🚀 )
```
I guess that we can probably use `reprise`, although a bit more difficult to understand (as it can be interpreted as "Take back the print"), context should help (being around "Stop print"). Although, there's not really a way to shorten this further, if ever necessary. (Would it be worthwhile to take a look into being able to scroll menu items?)